File: rev-com-server/src/main/java/com/codergb/bean/book/Writer.java
package com.codergb.bean.book;

import com.codergb.bean.FileInfo;
import lombok.*;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper = true)
@ToString
public class Writer extends FileInfo {
  private String id;
  private String name;
  private String gender;
  private Date birth;
  private Date deadTime;
  private String birthPlace;
  private String area;
  private String foreignName;
  private String alias;
  private String description;
  private Date createTime;
  private Date updateTime;
}

File: rev-com-manage/src/types/book/writer.ts
import {IFileInfo} from "@/types/FileInfo";

interface IWriter extends IFileInfo{
  id:string,
  name:string,
  gender:string,
  birth:string,
  deadTime:string,
  birthPlace:string,
  area:string,
  foreignName:string,
  alias:string,
  description:string,
  originalName:string,
  createTime:string,
  updateTime:string,
}
export type{
  IWriter
}
